database/dbtx.go

```diff
@@ -0,0 +1,81 @@
+package database
+
+import (
+	"github.com/go-pg/pg/v9"
+	"github.com/go-pg/pg/v9/orm"
+)
+
+// DBTx is an interface type implemented both by pg.DB and pg.Tx.
+// We use DBTx to execute db operations with or without transaction context.
+type DBTx interface {
```

Comment:
This interface is not necessarily implemented by TX, it's also implemented by just DB.
Please find a more suitable name